virttest/utils_test/libvirt.py

```diff
@@ -3413,3 +3415,111 @@ def check_machine_type_arch(machine_type):
                                        "supported on the host with "
                                        "arch '%s'" % (machine_type,
                                                       arch))
+
+
+def setup_libvirtd_conf_dict(params):
+    """
+    Read and set the required parameters to dict
+
+    :param params: the parameters to be used
+    :return: a dict that includes required parameters
+    """
+    conf_dict = {}
+
+    conf_dict['keepalive_interval'] = int(params.get("keepalive_interval", '5'))
```

Comment:
those keyes are hardcoded, not generic,suggest move to specific test case

Reply:
Done